app/pb/annotations.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
+
+package org.couchers;
+
+import "google/protobuf/descriptor.proto";
+
+extend google.protobuf.FieldOptions {
+  // whether the field is sensitive and should not be logged
+  bool sensitive = 50000;
```

Comment:
What does the number indicate here? (Too lazy to look up docs)

Reply:
It's the field number like elsewhere in protos. The "internal extension" fields should be between 50000-59999 according to docs.

(There's some common/public annotations e.g. the http transcoding stuff that needs to be unique, so there's some registry of those as well)

Reply:
But it is never seen on the wire?

Reply:
We never see it on the wire. But protobuf eats protobuf: all the code generation API surfaces (it's all plugin based) pass around these descriptor protos, etc.

Reply:
And it doesn't use strings for that? silly. Anyway, not our fault :) resolving.